pywt/_cwt.py

```diff
@@ -11,6 +8,20 @@
 __all__ = ["cwt"]
 
 
+import numpy as np
+
+try:
+    from scipy.fftpack import next_fast_len
+except ImportError:
+    # Do provide a fallback so scipy is an optional requirement
+    def next_fast_len(n):
+        """Given a number of samples `n`, returns the next power of two
+        following this numbe to take advantage of FFT speedup.
+        This fallback is less efficient as `scipy.fftpack.next_fast_len`
```

Comment:
minor typos:

numbe -> number
"less efficient as" -> "less efficient than"

Reply:
Hi, thanks for the typos, and btw, congrats for having taken your project to google summer camp.
I looked for pywavelet on the google camp site but could not find it.

In which section is it?

Reply:
The GSOC project I was referring to is for SciPy itself rather than PyWavelets. The various Python projects planning to participate in GSOC are listed at:
https://blogs.python-gsoc.org/en/project-ideas/

and the specific project I mentioned is the fftpack one described here:
https://github.com/scipy/scipy/wiki/GSoC-2019-project-ideas

At this point SciPy has received final proposals from potential students, but we won't know until May 6th whether our proposal has been accepted by Google.